views/signup.handlebars

```diff
@@ -19,11 +8,8 @@
                             href="/">Sign In</a></div>
                 </div>
                 <div class="panel-body">
-                    <form class="form-horizontal" role="form" method="post" action="/signup"
-                        onSubmit="return checkblank(this);">
-                        <% if (message.length > 0) { %>
-                        <div class="alert alert-success col-sm-12"><%= message %></div>
-                        <% } %>
+                    <form class="form-horizontal" role="form" method="post" action="/signup" onSubmit="return checkblank(this);">
+                        <div class="alert alert-success col-sm-12">Please fill out the form below:</div>
```

Comment:
I just replaced the non-handlebars templating stuff with a standard message.  Feel free to modify.